en/docs/learn/references/natural-functions.md

```diff
@@ -119,9 +119,9 @@ Follow the steps below to implement the integration.
 5. The response will be a review of the blog content with the category and rating.
     ```json
```

Comment:
Add empty new line before and after of each code block and image links. check all the places

Reply:
Updated with https://github.com/wso2/docs-bi/commit/8343ebaad98ab71d70c5222c943ff96bb5362608